proto/folder.rs

```diff
@@ -28,7 +28,11 @@ pub(crate) fn read_proto_folder(folder_path: PathBuf) -> io::Result<ProtoFolder>
             let path = entry.path();
             if path.is_dir() {
                 folders.push(path);
-            } else if path.extension().unwrap() == "proto" {
+            } else if path.is_file()
+                && path.file_name().unwrap() != ".DS_Store"
```

Comment:
What extension does this file have? If it is not ".proto" than this code should already work appropriately.

Reply:
Can you please check current `main` branch if this error still exists. I've removed usage of `unwrap`, so the problem should be fixed, I think.

Thanks for your contribution!!!!!!!

Reply:
It's work fine, Happy to help